src/xcode/ENA/ENA/Source/Extensions/UIImage+Resize.swift

```diff
@@ -0,0 +1,30 @@
+////
+// 🦠 Corona-Warn-App
+//
+
+import Foundation
+import UIKit
+
+extension UIImage {
+
+	/*** Resize and create a new UIImage with targetSize and keep the aspect ratio of the original image*/
```

Comment:
Just wondering, what /*** means ;-) Is this a multiline comment? Shouldn't it be like this?

`// Resize and create a new UIImage with targetSize and keep the aspect ratio of the original`.